docs/habana-gaudi.md

```diff
@@ -142,11 +142,11 @@ The [`SFTTrainer`](https://huggingface.co/docs/trl/sft_trainer) from `trl` does
 
 ## Install and run InstructLab with Intel Gaudi
 
-Install `InstructLab` from checkout with additional dependency `habana`:
+Install `InstructLab` from checkout with additional dependencies:
 
 ```shell
 . $HABANALABS_VIRTUAL_DIR/bin/activate
-pip install './instructlab[habana]'
+pip install -r instructlab/requirements-hpu.txt ./instructlab
```

Comment:
`instructlab/requirements.txt` is not required?

Reply:
`pip install ./path` creates a wheel for the package in `path` and installs it. Our package use setuptool's dynamic dependencies to define dependencies for the `instructlab` wheel. The dynamic dependencies are generated from `requirements.txt`.

In fewer words: `pip install instructlab` installs everything from `requirements.txt`.